fabtests/scripts/runfabtests.sh

```diff
@@ -619,10 +622,12 @@ function usage {
 	errcho -e " -u\tconfigure option for complex tests"
 	errcho -e " -T\ttimeout value in seconds"
 	errcho -e " -S\tStrict mode: -FI_ENODATA, -FI_ENOSYS errors would be treated as failures instead of skipped/notrun"
+	errcho -e " -C\Additional client test arguments: Parameters to pass to client fabtests"
+	errcho -e " -A\Additional server test arguments: Parameters to pass to server fabtests"
```

Comment:
nit: Can we use -L (for listener) or P (for passive) in place of A?

Reply:
Sure, I've changed it to -L.